atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/backup/transaction/Transactions3TableInteraction.java

```diff
@@ -123,7 +124,6 @@ public Statement bindAbortStatement(PreparedStatement preparedAbortStatement, Tr
         BoundStatement bound = preparedAbortStatement.bind(rowKeyBb, columnNameBb, valueBb);
         return bound.setConsistencyLevel(ConsistencyLevel.QUORUM)
                 .setSerialConsistencyLevel(ConsistencyLevel.SERIAL)
-                .setDefaultTimestamp(CellValuePutter.SET_TIMESTAMP + 1)
```

Comment:
From javadoc of `Statement#setDefaultTimestamp(long)`:

```
   * <p>Note that unlike other configuration, when this statement is prepared {@link
   * BoundStatement}s created off of {@link PreparedStatement} do not inherit this configuration.
```

Reply:
It doesn't seem like this should apply, given that we set it on the bound statement already